aws/resource_aws_codebuild_project_test.go

```diff
@@ -2002,6 +2023,31 @@ resource "aws_codebuild_project" "test" {
 `, description, rName)
 }
 
+func testAccAWSCodeBuildProjectConfig_SourceVersion(rName, sourceVersion string) string {
+	return testAccAWSCodeBuildProjectConfig_Base_ServiceRole(rName) + fmt.Sprintf(`
+resource "aws_codebuild_project" "test" {
+  name           = "%s"
+  service_role   = "${aws_iam_role.test.arn}"
+
+  artifacts {
+    type = "NO_ARTIFACTS"
+  }
+
+  environment {
+    compute_type = "BUILD_GENERAL1_SMALL"
+    image        = "2"
+    type         = "LINUX_CONTAINER"
+  }
+
+  source_version = "%s"
+  source {
+    type     = "GITHUB"
+    location = "https://github.com/hashicorp/packer.git"
+  }
+}
+`, sourceVersion, rName)
```

Comment:
The new test is currently failing because these parameters are switched 😄 

```
--- FAIL: TestAccAWSCodeBuildProject_SourceVersion (36.38s)
    testing.go:640: Step 0 error: Check failed: Check 2/2 error: aws_codebuild_project.test: Attribute 'source_version' expected "master", got "tf-acc-test-2042168198443316982"
```

```suggestion
`, rName, sourceVersion)
```